common/test/uiComponent/core/RenderingActorTest.scala

```diff
@@ -0,0 +1,49 @@
+package uiComponent.core
+
+import akka.actor.{ActorSystem, Props}
+import akka.pattern.ask
+import akka.util.Timeout
+import org.scalatest.{DoNotDiscover, FlatSpec, Matchers}
+import play.api.libs.json.{JsValue, Json}
+import test.{ConfiguredTestSuite, WithTestContext}
+import uiComponent.UIComponent
+
+import scala.concurrent.Await
+import scala.concurrent.duration._
+import scala.util.{Failure, Success, Try}
+
+@DoNotDiscover class RenderingActorTest
+  extends FlatSpec
+  with ConfiguredTestSuite
+  with WithTestContext
+  with Matchers {
+
+  lazy val actorSystem: ActorSystem = app.actorSystem
+  implicit lazy val timeout = new Timeout(2.seconds)
+
+  class TestRenderingActor extends RenderingActor {
+    override def javascriptFile: String = "components/TestButtonComponent.js"
+  }
+
+  lazy val actor = actorSystem.actorOf(Props(new TestRenderingActor))
+
+  "Sending rendering message" should "return correct string" in {
```

Comment:
Misleading name of test. Should be changed to `Sending rendering message should return a string`